src/Discord.Net.Core/Entities/Activities/Game.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System.Diagnostics;
+using System.Diagnostics;
 
 namespace Discord
 {
     [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
     public class Game : IActivity
     {
+        internal Game() { }
```

Comment:
Why move the ctor?

Reply:
Mainly because I saw other files like `GameAsset`'s ctor was at front. I'll change that back though.